app/commands/user/invalidate_avatar_in_cloudfront.rb

```diff
@@ -0,0 +1,36 @@
+class User::InvalidateAvatarInCloudfront
```

Comment:
This isn't yet used, correct?

Reply:
Used here: https://github.com/exercism/website/pull/5980/files#diff-aee3db72f1aac6b9ee5196b908407de2b4cd53e11ae655c10ee2aa401b2a8255R12